ropgadget/loaders/elf.py

```diff
@@ -340,6 +341,8 @@ def getArch(self):
             return CS_ARCH_PPC
         elif self.__ElfHeader.e_machine == ELFFlags.EM_SPARCv8p:
             return CS_ARCH_SPARC
+        elif self.__ElfHeader.e_machine == ELFFlags.EM_RISCV:
```

Comment:
So, there are no separate flags for riscv 32 and 64?

Reply:
yes